src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlFloat.java

```diff
@@ -28,8 +28,11 @@ public Object createdata() {
         // for float in SQL Server, any precision <=24 is considered as real so the value must be within
         // SqlTypeValue.REAL.minValue/maxValue
         if (precision > 24) {
+            minvalue = ((Double) minvalue < Double.MIN_VALUE) ? Double.MIN_VALUE : minvalue;
+            maxvalue = ((Double) maxvalue > Double.MAX_VALUE) ? Double.MAX_VALUE : maxvalue;

```

Comment:
the reason I don't want to change the value in SqlTypeValue is because that is the [actual range of the data type](https://docs.microsoft.com/en-us/sql/t-sql/data-types/float-and-real-transact-sql?view=sql-server-ver15). I'll add a comment here instead